sources/packages/web/src/types/contracts/student/ApplicationContract.ts

```diff
@@ -76,14 +76,20 @@ export interface ApplicationStatusToBeUpdatedDto {
   applicationStatus: ApplicationStatus;
 }
 
+/**
+ * DTO with primary identifiers of application.
+ */
+export interface ApplicationPrimaryDTO {
+  id: number;
+  applicationNumber: string;
+}
+
 /**
  * Base DTO for application dynamic data
  */
-export interface GetApplicationBaseDTO {
+export interface GetApplicationBaseDTO extends ApplicationPrimaryDTO {
```

Comment:
I am missing why `GetApplicationBaseDTO` is now extending the `ApplicationPrimaryDTO`. Can you please clarify it?